test/text-field.html

```diff
@@ -55,6 +55,37 @@
 
     });
 
+    describe('slots', () => {
+      let textField;
+
+      function afterFlush() {
+        return new Promise(resolve => {
+          if (window.ShadyDOM) {
+            window.ShadyDOM.flush();
+            resolve();
+          } else {
+            requestAnimationFrame(resolve);
+          }
+        });
+      }
+
+      beforeEach(function() {
+        textField = fixture('default');
+      });
+
+      it('should not break when adding unexpected content to input slot', done => {
+        const div = document.createElement('div');
+        div.setAttribute('slot', 'input');
+        textField.appendChild(div);
+        const errorSpy = sinon.spy();
+        window.onerror = errorSpy;
+        afterFlush().then(() => {
+          expect(errorSpy).to.not.be.called;
```

Comment:
Should probably call `errorSpy.restore();` before the expect so `window.onerror` doesn't remain patched

Reply:
Done. For some reason there is no `restore()` method, so re-assigned `onerror` manually.